lib/spack/spack/bootstrap/core.py

```diff
@@ -465,14 +465,20 @@ def clingo_root_spec() -> str:
     return _root_spec("clingo-bootstrap@spack+python")
 
 
+def file_root_spec() -> str:
+    """Return the root spec used to bootstrap file"""
+    return _root_spec("file")
```

Comment:
Should this be `"win-file" if IS_WINDOWS else "file"`?

Reply:
Right we should do it for both, good catch.